include/utils.hpp

```diff
@@ -37,9 +37,113 @@ namespace utils {
+                            1.80,
+                            2.06,
+                            2.34,
+                            2.65,
+                            2.98,
+                            3.34,
+                            3.72,
+                            4.14,
+                            4.58,
+                            5.06,
+                            5.57,
+                            6.11,
+                            6.68,
+                            7.30,
+                            7.94,
+                            8.63,
+                            9.35,
+                            10.12,
+                            10.92,
+                            11.77,
+                            12.66,
+                            13.60,
+                            14.59,
+                            15.62,
+                            16.70,
+                            17.83,
+                            19.02,
+                            20.26,
+                            21.55,
+                            22.90,
+                            24.30,
+                            25.77,
+                            27.23,
+                            28.70,
+                            30.16,
+                            31.63,
+                            33.09,
+                            34.56,
+                            36.02,
+                            37.49,
+                            38.95,
+                            40.41,
+                            41.88,
+                            43.34,
+                            44.81,
+                            46.27,
+                            47.74,
+                            49.20,
+                            50.67,
+                            52.13,
+                            53.60,
+                            55.06,
+                            56.53,
+                            57.99,
+                            59.45,
+                            60.92,
+                            62.38,
+                            63.85,
+                            65.31,
+                            66.78,
+                            68.24,
+                            69.71,
+                            71.17,
+                            72.64,
+                            74.10,
+                            75.57,
+                            77.03,
+                            78.50,
+                            79.96,
+                            81.42,
+                            82.89,
+                            84.35,
+                            85.82,
+                            87.28,
+                            88.75,
+                            90.21,
+                            91.68,
+                            93.14,
+                            94.61,
+                            96.07,
+                            97.54,
+                            99.00
+    };
+
+    float calc = (current * 100) / (float)total;
```

Comment:
this should be converted to an [int] and then the [int] is simply used as a lookup?!

No need to iterate through this array.

Reply:
in fact... that was the point of the exercise, I think?

We use the rank from 0-99 but instead of returning it, we return the value in the array on that index?!

What that if... statement is doing is ... something else entirely, I think

Reply:
I get it, I was trying to see if the returned value was below the number indicated by coef to return the coef, I get that it should be the oposite.